src/ajax/delete_element.php

```diff
@@ -0,0 +1,31 @@
+<?php
+require "../connexion.php";
+
+session_start();
+Connexion::set_up_connection("../../res/");
+
+if (isset($_POST["id"], $_POST["mode"]) && $_SESSION["role"] == 1) {
+    $id = $_POST["id"];
+    $mode = $_POST["mode"];
+    $query = "";
+
+    if ($mode == 0)
+        $query = deleteAccount();
+    else if ($mode == 1)
+        $query = deleteImage();
```

Comment:
```suggestion
    $query = ($mode == 0) ? deleteAccount() : deleteImage();
```
This is like a boolean condition, you can write this condition with only one line

Reply:
can have more modes so it should stay with if else